Cvar_SetValue is called from source/game/g_unlagged.c, so declare the
function in a shared header file that is included in all modules.

diff -ur alienarena-7.71.2-svn5674.orig/source/game/q_shared.h alienarena-7.71.2-svn5674/source/game/q_shared.h
--- alienarena-7.71.2-svn5674.orig/source/game/q_shared.h	2022-12-17 16:30:21.329335731 +0100
+++ alienarena-7.71.2-svn5674/source/game/q_shared.h	2022-12-17 16:48:33.500972498 +0100
@@ -395,6 +395,9 @@
 #define CVARDOC_FLOAT	2048
 #define CVARDOC_INT		4096
 
+// expands value to a string and calls Cvar_Set
+void	Cvar_SetValue (const char *var_name, float value);
+
 // nothing outside the Cvar_*() functions should modify these fields!
 typedef struct cvar_s
 {
diff -ur alienarena-7.71.2-svn5674.orig/source/qcommon/qcommon.h alienarena-7.71.2-svn5674/source/qcommon/qcommon.h
--- alienarena-7.71.2-svn5674.orig/source/qcommon/qcommon.h	2022-12-17 16:30:21.329335731 +0100
+++ alienarena-7.71.2-svn5674/source/qcommon/qcommon.h	2022-12-17 16:48:29.782010695 +0100
@@ -540,9 +540,6 @@
 
 void	Cvar_FullSet (const char *var_name, const char *value, int flags);
 
-// expands value to a string and calls Cvar_Set
-void	Cvar_SetValue (const char *var_name, float value);
-
 // returns 0 if not defined or non numeric
 float	Cvar_VariableValue (const char *var_name);
 
